Assets/Minigames/Fight/Scripts/Entity/Player/PlayerAOE.cs

```diff
@@ -0,0 +1,46 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Minigames.Fight
+{
+    public class PlayerAOE : AOEController
+    {
+        protected override void OnTriggerEnter2D(Collider2D collision)
+        {
+            base.OnTriggerEnter2D(collision);
+            if (collision.gameObject.layer == PhysicsUtils.EnemyLayer)
+            {
+                Entity entity = collision.GetComponent<Entity>();
+
+                effectedEntities.Add(entity);
+            }
+        }
+        protected override void OnTriggerExit2D(Collider2D collision)
```

Comment:
feels like there is a LOT of overlap in the AOE controllers, could we re-use some more code?